modules/images-other-jenkins-oauth-auth.adoc

```diff
@@ -18,26 +18,27 @@ Jenkins supports both browser and non-browser access.
 
 Valid users are automatically added to the Jenkins authorization matrix at log
 in, where {product-title} `Roles` dictate the specific Jenkins permissions the
-users have.  The `Roles` used by default are the `admin`, `edit`, and `view` 
-`Roles` in the `Project` or namespace that Jenkins is running in.
+users have.  The `Roles` used by default are the predefined `admin`, `edit`, and `view`.
+The login plugins executes self-SAR requests against those `Roles` in the `Project` 
+or namespace that Jenkins is running in.
 
 Users with the `admin` role have the traditional Jenkins administrative
 user permissions. Users with the `edit` or `view` role have progressively
 fewer permissions.
 
-Those defaults noted, the which set of {product-title} `Roles` and which Jenkins permissions
+The default {product-title} `admin`, `edit`, and `view` `Roles` and the Jenkins permissions
 those `Roles` are assigned in the Jenkins instance are configurable.
 
-The login plugin will look for a `ConfigMap` named `openshift-jenkins-login-plugin-config`. Specifically, when running Jenkins in an {product-title} `Pod`, it will look in the namespace that Jenkins is running in. Otherwise, it looks in the namespace specified in the "client ID" as explained in "Secondary Scenarios" down below.
+When running Jenkins in an {product-title} `Pod`, the login plug-in looks for a `ConfigMap` named `openshift-jenkins-login-plugin-config` in the namespace that Jenkins is running in. 
 
-If this plugin finds and can read in that ConfigMap, it then:
+If this plugin finds and can read in that `ConfigMap`, you can define the `Role` to Jenkins Permission mappings.  Specifically:
 
- * Treats the key/value pairs in the ConfigMap as Jenkins permission to OpenShift Role mappings.
- * The key is the Jenkins permission group short ID and the Jenkins permission short ID, with those two separated by a hyphen or - character.
- * So if you want to add the Overall Jenkins Administer permission to an OpenShift Role, the key should be Overall-Administer
+ * The login plugin treats the key and value pairs in the `ConfigMap` as Jenkins permission to OpenShift Role mappings.
+ * The key is the Jenkins permission group short ID and the Jenkins permission short ID, with those two separated by a hyphen character.
+ * If you want to add the `Overall Jenkins Administer` permission to an {product-title} `Role`, the key should be `Overall-Administer`
  * To get a sense of which permission groups and permissions IDs are available, go the the matrix authorization page in the Jenkins console and IDs for the groups and individual permissions in the table they provide.
```

Comment:
I didn't notice before, but this part may actually explain how users using a locale different from "en" should create their configMap.

Reply:
I don't have other comments @gabemontero , looks good to me.